pkg/bpmn_engine/variable_scope/scope.go

```diff
@@ -0,0 +1,79 @@
+package variable_scope
+
+type VarScope interface {
```

Comment:
I like you put this into a separate package.
please don't use Abbreviations, but rather "VariableContext" instead.

Reply:
Good idea.